cmd/ak/cmd/triggers/create.go

```diff
@@ -32,30 +33,47 @@ var createCmd = common.StandardCommand(&cobra.Command{
 			return common.NewExitCodeError(common.NotFoundExitCode, err)
 		}
 
-		c, cid, err := r.ConnectionNameOrID(connection, "")
-		if err != nil {
-			if errors.As(err, resolver.NotFoundErrorType) {
-				err = common.NewExitCodeError(common.NotFoundExitCode, err)
-			}
-			return err
-		}
-		if !c.IsValid() {
-			err = fmt.Errorf("connection %q not found", connection)
-			return common.NewExitCodeError(common.NotFoundExitCode, err)
-		}
-
 		cl, err := sdktypes.ParseCodeLocation(loc)
 		if err != nil {
 			return fmt.Errorf("invalid entry-point location %q: %w", loc, err)
 		}
 
+		connectionID := ""
+		data := make(map[string]sdktypes.Value)
+		if cmd.Flags().Changed("schedule") {
+			if connection != "" || event != "" {
+				return fmt.Errorf(`flags(s) "connection", "event" are not compatible with "schedule"`)
```

Comment:
As much as possible, you shouldn't write custom logic for this, you should use Cobra's flag modifiers. See `MarkFlagsMutuallyExclusive` and `MarkFlagsRequiredTogether`.